src/store/plugins/ledgerConnection.js

```diff
@@ -3,6 +3,7 @@ import Ae from '@aeternity/ledger-app-api';
 import { Crypto } from '@aeternity/aepp-sdk/es';
 
 export default async (store) => {
+  await TransportU2F.isSupported().then(supported => store.commit('setLedgerSupported', supported));
```

Comment:
I mean something like this:
```suggestion
  const isSupported = await TransportU2F.isSupported();
  store.commit('setLedgerSupported', isSupported);
  if (!isSupported) return;
```
`.then` syntax is not necessary here.